problems/problem_3.md

```diff
@@ -0,0 +1,7 @@
+# I want to refactor a legacy system. You want to rewrite it from scratch. Argument. Then, switch our roles
+
+## Argument in favour of rewrite:
+### The cost to find, train and employ developers to work on the legacy system is very high. We can probably save money on maintenance once we actually rewrite the whole thing. We can even go with a more popular stack, with a more active community around. We'll have a bigger pool of developers to choose from. Since we know the flaws and limitations of the old system, we can fix all that with a rewrite. And this time, we'll document it properly.
+
+## Argument in favour of refactor:
+### While it might be cheaper to hire developers for the more popular stack but you didn't take into account the cost to build the product from the ground up. By refactoring we can keep our codebase regularly updated. We can pick and choose what we want to refactor and we won't have the pressure to rewrite from stack. Even if we go with rewriting we'll still have to maintain the legacy system till our rewrite is completed. And by rewriting we'll essentially throw away a lot of bug fixes that were noticed during production. 
```

Comment:
In the last statement are you concluding in the favour of a rewrite? :D

Reply:
In the last statement, I'm mentioning a con of rewriting.